BaseApi/Transformer/AbstractTransformer.php

```diff
@@ -81,6 +83,28 @@ protected function hasGroup($group)
      *
      * @return FacadeInterface
      */
+    public function cacheTransform($mixed)
+    {
+        $isObject = is_object($mixed);
```

Comment:
if isn't a object you can test here if not return $this->transform($mixed); and you can remove other if